ui/components/component-library/button-base/button-base.scss

```diff
@@ -1,4 +1,4 @@
-.mm-button {
+.mm-button-base {
   position: relative;
   height: 40px;
   padding: 0;
```

Comment:
Could things like padding, border-radius, color, background-color all be `Box` props? Then it can be overwritten by the Box prop in variant components and it saves CSS here and then also CSS that it takes to overwrite this CSS in the variant component.